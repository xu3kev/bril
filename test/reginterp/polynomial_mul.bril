main{
a_0:int = const 97;
a_1:int = const 33;
a_2:int = const 23;
a_3:int = const 78;
a_4:int = const 39;
a_5:int = const 0;
a_6:int = const 12;
a_7:int = const 2;
a_8:int = const 84;
a_9:int = const 62;
a_10:int = const 36;
a_11:int = const 12;
a_12:int = const 90;
a_13:int = const 32;
a_14:int = const 70;
b_0:int = const 4;
b_1:int = const 85;
b_2:int = const 79;
b_3:int = const 89;
b_4:int = const 65;
b_5:int = const 83;
b_6:int = const 55;
b_7:int = const 56;
b_8:int = const 41;
b_9:int = const 94;
b_10:int = const 94;
b_11:int = const 74;
b_12:int = const 94;
b_13:int = const 41;
b_14:int = const 72;
c_0:int = const 0;
c_1:int = const 0;
c_2:int = const 0;
c_3:int = const 0;
c_4:int = const 0;
c_5:int = const 0;
c_6:int = const 0;
c_7:int = const 0;
c_8:int = const 0;
c_9:int = const 0;
c_10:int = const 0;
c_11:int = const 0;
c_12:int = const 0;
c_13:int = const 0;
c_14:int = const 0;
c_15:int = const 0;
c_16:int = const 0;
c_17:int = const 0;
c_18:int = const 0;
c_19:int = const 0;
c_20:int = const 0;
c_21:int = const 0;
c_22:int = const 0;
c_23:int = const 0;
c_24:int = const 0;
c_25:int = const 0;
c_26:int = const 0;
c_27:int = const 0;
c_28:int = const 0;
tmp:int = mul a_0 b_0;
c_0:int = add c_0 tmp;
tmp:int = mul a_0 b_1;
c_1:int = add c_1 tmp;
tmp:int = mul a_0 b_2;
c_2:int = add c_2 tmp;
tmp:int = mul a_0 b_3;
c_3:int = add c_3 tmp;
tmp:int = mul a_0 b_4;
c_4:int = add c_4 tmp;
tmp:int = mul a_0 b_5;
c_5:int = add c_5 tmp;
tmp:int = mul a_0 b_6;
c_6:int = add c_6 tmp;
tmp:int = mul a_0 b_7;
c_7:int = add c_7 tmp;
tmp:int = mul a_0 b_8;
c_8:int = add c_8 tmp;
tmp:int = mul a_0 b_9;
c_9:int = add c_9 tmp;
tmp:int = mul a_0 b_10;
c_10:int = add c_10 tmp;
tmp:int = mul a_0 b_11;
c_11:int = add c_11 tmp;
tmp:int = mul a_0 b_12;
c_12:int = add c_12 tmp;
tmp:int = mul a_0 b_13;
c_13:int = add c_13 tmp;
tmp:int = mul a_0 b_14;
c_14:int = add c_14 tmp;
tmp:int = mul a_1 b_0;
c_1:int = add c_1 tmp;
tmp:int = mul a_1 b_1;
c_2:int = add c_2 tmp;
tmp:int = mul a_1 b_2;
c_3:int = add c_3 tmp;
tmp:int = mul a_1 b_3;
c_4:int = add c_4 tmp;
tmp:int = mul a_1 b_4;
c_5:int = add c_5 tmp;
tmp:int = mul a_1 b_5;
c_6:int = add c_6 tmp;
tmp:int = mul a_1 b_6;
c_7:int = add c_7 tmp;
tmp:int = mul a_1 b_7;
c_8:int = add c_8 tmp;
tmp:int = mul a_1 b_8;
c_9:int = add c_9 tmp;
tmp:int = mul a_1 b_9;
c_10:int = add c_10 tmp;
tmp:int = mul a_1 b_10;
c_11:int = add c_11 tmp;
tmp:int = mul a_1 b_11;
c_12:int = add c_12 tmp;
tmp:int = mul a_1 b_12;
c_13:int = add c_13 tmp;
tmp:int = mul a_1 b_13;
c_14:int = add c_14 tmp;
tmp:int = mul a_1 b_14;
c_15:int = add c_15 tmp;
tmp:int = mul a_2 b_0;
c_2:int = add c_2 tmp;
tmp:int = mul a_2 b_1;
c_3:int = add c_3 tmp;
tmp:int = mul a_2 b_2;
c_4:int = add c_4 tmp;
tmp:int = mul a_2 b_3;
c_5:int = add c_5 tmp;
tmp:int = mul a_2 b_4;
c_6:int = add c_6 tmp;
tmp:int = mul a_2 b_5;
c_7:int = add c_7 tmp;
tmp:int = mul a_2 b_6;
c_8:int = add c_8 tmp;
tmp:int = mul a_2 b_7;
c_9:int = add c_9 tmp;
tmp:int = mul a_2 b_8;
c_10:int = add c_10 tmp;
tmp:int = mul a_2 b_9;
c_11:int = add c_11 tmp;
tmp:int = mul a_2 b_10;
c_12:int = add c_12 tmp;
tmp:int = mul a_2 b_11;
c_13:int = add c_13 tmp;
tmp:int = mul a_2 b_12;
c_14:int = add c_14 tmp;
tmp:int = mul a_2 b_13;
c_15:int = add c_15 tmp;
tmp:int = mul a_2 b_14;
c_16:int = add c_16 tmp;
tmp:int = mul a_3 b_0;
c_3:int = add c_3 tmp;
tmp:int = mul a_3 b_1;
c_4:int = add c_4 tmp;
tmp:int = mul a_3 b_2;
c_5:int = add c_5 tmp;
tmp:int = mul a_3 b_3;
c_6:int = add c_6 tmp;
tmp:int = mul a_3 b_4;
c_7:int = add c_7 tmp;
tmp:int = mul a_3 b_5;
c_8:int = add c_8 tmp;
tmp:int = mul a_3 b_6;
c_9:int = add c_9 tmp;
tmp:int = mul a_3 b_7;
c_10:int = add c_10 tmp;
tmp:int = mul a_3 b_8;
c_11:int = add c_11 tmp;
tmp:int = mul a_3 b_9;
c_12:int = add c_12 tmp;
tmp:int = mul a_3 b_10;
c_13:int = add c_13 tmp;
tmp:int = mul a_3 b_11;
c_14:int = add c_14 tmp;
tmp:int = mul a_3 b_12;
c_15:int = add c_15 tmp;
tmp:int = mul a_3 b_13;
c_16:int = add c_16 tmp;
tmp:int = mul a_3 b_14;
c_17:int = add c_17 tmp;
tmp:int = mul a_4 b_0;
c_4:int = add c_4 tmp;
tmp:int = mul a_4 b_1;
c_5:int = add c_5 tmp;
tmp:int = mul a_4 b_2;
c_6:int = add c_6 tmp;
tmp:int = mul a_4 b_3;
c_7:int = add c_7 tmp;
tmp:int = mul a_4 b_4;
c_8:int = add c_8 tmp;
tmp:int = mul a_4 b_5;
c_9:int = add c_9 tmp;
tmp:int = mul a_4 b_6;
c_10:int = add c_10 tmp;
tmp:int = mul a_4 b_7;
c_11:int = add c_11 tmp;
tmp:int = mul a_4 b_8;
c_12:int = add c_12 tmp;
tmp:int = mul a_4 b_9;
c_13:int = add c_13 tmp;
tmp:int = mul a_4 b_10;
c_14:int = add c_14 tmp;
tmp:int = mul a_4 b_11;
c_15:int = add c_15 tmp;
tmp:int = mul a_4 b_12;
c_16:int = add c_16 tmp;
tmp:int = mul a_4 b_13;
c_17:int = add c_17 tmp;
tmp:int = mul a_4 b_14;
c_18:int = add c_18 tmp;
tmp:int = mul a_5 b_0;
c_5:int = add c_5 tmp;
tmp:int = mul a_5 b_1;
c_6:int = add c_6 tmp;
tmp:int = mul a_5 b_2;
c_7:int = add c_7 tmp;
tmp:int = mul a_5 b_3;
c_8:int = add c_8 tmp;
tmp:int = mul a_5 b_4;
c_9:int = add c_9 tmp;
tmp:int = mul a_5 b_5;
c_10:int = add c_10 tmp;
tmp:int = mul a_5 b_6;
c_11:int = add c_11 tmp;
tmp:int = mul a_5 b_7;
c_12:int = add c_12 tmp;
tmp:int = mul a_5 b_8;
c_13:int = add c_13 tmp;
tmp:int = mul a_5 b_9;
c_14:int = add c_14 tmp;
tmp:int = mul a_5 b_10;
c_15:int = add c_15 tmp;
tmp:int = mul a_5 b_11;
c_16:int = add c_16 tmp;
tmp:int = mul a_5 b_12;
c_17:int = add c_17 tmp;
tmp:int = mul a_5 b_13;
c_18:int = add c_18 tmp;
tmp:int = mul a_5 b_14;
c_19:int = add c_19 tmp;
tmp:int = mul a_6 b_0;
c_6:int = add c_6 tmp;
tmp:int = mul a_6 b_1;
c_7:int = add c_7 tmp;
tmp:int = mul a_6 b_2;
c_8:int = add c_8 tmp;
tmp:int = mul a_6 b_3;
c_9:int = add c_9 tmp;
tmp:int = mul a_6 b_4;
c_10:int = add c_10 tmp;
tmp:int = mul a_6 b_5;
c_11:int = add c_11 tmp;
tmp:int = mul a_6 b_6;
c_12:int = add c_12 tmp;
tmp:int = mul a_6 b_7;
c_13:int = add c_13 tmp;
tmp:int = mul a_6 b_8;
c_14:int = add c_14 tmp;
tmp:int = mul a_6 b_9;
c_15:int = add c_15 tmp;
tmp:int = mul a_6 b_10;
c_16:int = add c_16 tmp;
tmp:int = mul a_6 b_11;
c_17:int = add c_17 tmp;
tmp:int = mul a_6 b_12;
c_18:int = add c_18 tmp;
tmp:int = mul a_6 b_13;
c_19:int = add c_19 tmp;
tmp:int = mul a_6 b_14;
c_20:int = add c_20 tmp;
tmp:int = mul a_7 b_0;
c_7:int = add c_7 tmp;
tmp:int = mul a_7 b_1;
c_8:int = add c_8 tmp;
tmp:int = mul a_7 b_2;
c_9:int = add c_9 tmp;
tmp:int = mul a_7 b_3;
c_10:int = add c_10 tmp;
tmp:int = mul a_7 b_4;
c_11:int = add c_11 tmp;
tmp:int = mul a_7 b_5;
c_12:int = add c_12 tmp;
tmp:int = mul a_7 b_6;
c_13:int = add c_13 tmp;
tmp:int = mul a_7 b_7;
c_14:int = add c_14 tmp;
tmp:int = mul a_7 b_8;
c_15:int = add c_15 tmp;
tmp:int = mul a_7 b_9;
c_16:int = add c_16 tmp;
tmp:int = mul a_7 b_10;
c_17:int = add c_17 tmp;
tmp:int = mul a_7 b_11;
c_18:int = add c_18 tmp;
tmp:int = mul a_7 b_12;
c_19:int = add c_19 tmp;
tmp:int = mul a_7 b_13;
c_20:int = add c_20 tmp;
tmp:int = mul a_7 b_14;
c_21:int = add c_21 tmp;
tmp:int = mul a_8 b_0;
c_8:int = add c_8 tmp;
tmp:int = mul a_8 b_1;
c_9:int = add c_9 tmp;
tmp:int = mul a_8 b_2;
c_10:int = add c_10 tmp;
tmp:int = mul a_8 b_3;
c_11:int = add c_11 tmp;
tmp:int = mul a_8 b_4;
c_12:int = add c_12 tmp;
tmp:int = mul a_8 b_5;
c_13:int = add c_13 tmp;
tmp:int = mul a_8 b_6;
c_14:int = add c_14 tmp;
tmp:int = mul a_8 b_7;
c_15:int = add c_15 tmp;
tmp:int = mul a_8 b_8;
c_16:int = add c_16 tmp;
tmp:int = mul a_8 b_9;
c_17:int = add c_17 tmp;
tmp:int = mul a_8 b_10;
c_18:int = add c_18 tmp;
tmp:int = mul a_8 b_11;
c_19:int = add c_19 tmp;
tmp:int = mul a_8 b_12;
c_20:int = add c_20 tmp;
tmp:int = mul a_8 b_13;
c_21:int = add c_21 tmp;
tmp:int = mul a_8 b_14;
c_22:int = add c_22 tmp;
tmp:int = mul a_9 b_0;
c_9:int = add c_9 tmp;
tmp:int = mul a_9 b_1;
c_10:int = add c_10 tmp;
tmp:int = mul a_9 b_2;
c_11:int = add c_11 tmp;
tmp:int = mul a_9 b_3;
c_12:int = add c_12 tmp;
tmp:int = mul a_9 b_4;
c_13:int = add c_13 tmp;
tmp:int = mul a_9 b_5;
c_14:int = add c_14 tmp;
tmp:int = mul a_9 b_6;
c_15:int = add c_15 tmp;
tmp:int = mul a_9 b_7;
c_16:int = add c_16 tmp;
tmp:int = mul a_9 b_8;
c_17:int = add c_17 tmp;
tmp:int = mul a_9 b_9;
c_18:int = add c_18 tmp;
tmp:int = mul a_9 b_10;
c_19:int = add c_19 tmp;
tmp:int = mul a_9 b_11;
c_20:int = add c_20 tmp;
tmp:int = mul a_9 b_12;
c_21:int = add c_21 tmp;
tmp:int = mul a_9 b_13;
c_22:int = add c_22 tmp;
tmp:int = mul a_9 b_14;
c_23:int = add c_23 tmp;
tmp:int = mul a_10 b_0;
c_10:int = add c_10 tmp;
tmp:int = mul a_10 b_1;
c_11:int = add c_11 tmp;
tmp:int = mul a_10 b_2;
c_12:int = add c_12 tmp;
tmp:int = mul a_10 b_3;
c_13:int = add c_13 tmp;
tmp:int = mul a_10 b_4;
c_14:int = add c_14 tmp;
tmp:int = mul a_10 b_5;
c_15:int = add c_15 tmp;
tmp:int = mul a_10 b_6;
c_16:int = add c_16 tmp;
tmp:int = mul a_10 b_7;
c_17:int = add c_17 tmp;
tmp:int = mul a_10 b_8;
c_18:int = add c_18 tmp;
tmp:int = mul a_10 b_9;
c_19:int = add c_19 tmp;
tmp:int = mul a_10 b_10;
c_20:int = add c_20 tmp;
tmp:int = mul a_10 b_11;
c_21:int = add c_21 tmp;
tmp:int = mul a_10 b_12;
c_22:int = add c_22 tmp;
tmp:int = mul a_10 b_13;
c_23:int = add c_23 tmp;
tmp:int = mul a_10 b_14;
c_24:int = add c_24 tmp;
tmp:int = mul a_11 b_0;
c_11:int = add c_11 tmp;
tmp:int = mul a_11 b_1;
c_12:int = add c_12 tmp;
tmp:int = mul a_11 b_2;
c_13:int = add c_13 tmp;
tmp:int = mul a_11 b_3;
c_14:int = add c_14 tmp;
tmp:int = mul a_11 b_4;
c_15:int = add c_15 tmp;
tmp:int = mul a_11 b_5;
c_16:int = add c_16 tmp;
tmp:int = mul a_11 b_6;
c_17:int = add c_17 tmp;
tmp:int = mul a_11 b_7;
c_18:int = add c_18 tmp;
tmp:int = mul a_11 b_8;
c_19:int = add c_19 tmp;
tmp:int = mul a_11 b_9;
c_20:int = add c_20 tmp;
tmp:int = mul a_11 b_10;
c_21:int = add c_21 tmp;
tmp:int = mul a_11 b_11;
c_22:int = add c_22 tmp;
tmp:int = mul a_11 b_12;
c_23:int = add c_23 tmp;
tmp:int = mul a_11 b_13;
c_24:int = add c_24 tmp;
tmp:int = mul a_11 b_14;
c_25:int = add c_25 tmp;
tmp:int = mul a_12 b_0;
c_12:int = add c_12 tmp;
tmp:int = mul a_12 b_1;
c_13:int = add c_13 tmp;
tmp:int = mul a_12 b_2;
c_14:int = add c_14 tmp;
tmp:int = mul a_12 b_3;
c_15:int = add c_15 tmp;
tmp:int = mul a_12 b_4;
c_16:int = add c_16 tmp;
tmp:int = mul a_12 b_5;
c_17:int = add c_17 tmp;
tmp:int = mul a_12 b_6;
c_18:int = add c_18 tmp;
tmp:int = mul a_12 b_7;
c_19:int = add c_19 tmp;
tmp:int = mul a_12 b_8;
c_20:int = add c_20 tmp;
tmp:int = mul a_12 b_9;
c_21:int = add c_21 tmp;
tmp:int = mul a_12 b_10;
c_22:int = add c_22 tmp;
tmp:int = mul a_12 b_11;
c_23:int = add c_23 tmp;
tmp:int = mul a_12 b_12;
c_24:int = add c_24 tmp;
tmp:int = mul a_12 b_13;
c_25:int = add c_25 tmp;
tmp:int = mul a_12 b_14;
c_26:int = add c_26 tmp;
tmp:int = mul a_13 b_0;
c_13:int = add c_13 tmp;
tmp:int = mul a_13 b_1;
c_14:int = add c_14 tmp;
tmp:int = mul a_13 b_2;
c_15:int = add c_15 tmp;
tmp:int = mul a_13 b_3;
c_16:int = add c_16 tmp;
tmp:int = mul a_13 b_4;
c_17:int = add c_17 tmp;
tmp:int = mul a_13 b_5;
c_18:int = add c_18 tmp;
tmp:int = mul a_13 b_6;
c_19:int = add c_19 tmp;
tmp:int = mul a_13 b_7;
c_20:int = add c_20 tmp;
tmp:int = mul a_13 b_8;
c_21:int = add c_21 tmp;
tmp:int = mul a_13 b_9;
c_22:int = add c_22 tmp;
tmp:int = mul a_13 b_10;
c_23:int = add c_23 tmp;
tmp:int = mul a_13 b_11;
c_24:int = add c_24 tmp;
tmp:int = mul a_13 b_12;
c_25:int = add c_25 tmp;
tmp:int = mul a_13 b_13;
c_26:int = add c_26 tmp;
tmp:int = mul a_13 b_14;
c_27:int = add c_27 tmp;
tmp:int = mul a_14 b_0;
c_14:int = add c_14 tmp;
tmp:int = mul a_14 b_1;
c_15:int = add c_15 tmp;
tmp:int = mul a_14 b_2;
c_16:int = add c_16 tmp;
tmp:int = mul a_14 b_3;
c_17:int = add c_17 tmp;
tmp:int = mul a_14 b_4;
c_18:int = add c_18 tmp;
tmp:int = mul a_14 b_5;
c_19:int = add c_19 tmp;
tmp:int = mul a_14 b_6;
c_20:int = add c_20 tmp;
tmp:int = mul a_14 b_7;
c_21:int = add c_21 tmp;
tmp:int = mul a_14 b_8;
c_22:int = add c_22 tmp;
tmp:int = mul a_14 b_9;
c_23:int = add c_23 tmp;
tmp:int = mul a_14 b_10;
c_24:int = add c_24 tmp;
tmp:int = mul a_14 b_11;
c_25:int = add c_25 tmp;
tmp:int = mul a_14 b_12;
c_26:int = add c_26 tmp;
tmp:int = mul a_14 b_13;
c_27:int = add c_27 tmp;
tmp:int = mul a_14 b_14;
c_28:int = add c_28 tmp;
print c_0;
print c_1;
print c_2;
print c_3;
print c_4;
print c_5;
print c_6;
print c_7;
print c_8;
print c_9;
print c_10;
print c_11;
print c_12;
print c_13;
print c_14;
print c_15;
print c_16;
print c_17;
print c_18;
print c_19;
print c_20;
print c_21;
print c_22;
print c_23;
print c_24;
print c_25;
print c_26;
print c_27;
print c_28;
}
