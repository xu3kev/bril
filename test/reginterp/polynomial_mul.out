388
8377
10560
13507
17845
21720
19640
18725
17553
27900
32684
34432
38681
43493
43825
43117
33956
37185
32112
29210
27274
26900
27200
23316
20492
12742
14372
5174
5040
#load: 929
#store: 509
